### PR TITLE
refactor(devtools): migrate to standalone

### PR DESCRIPTION
**Pull request by JeanMeche**
Migrated with the schematics and cleanup by hand.

### CODE REVIEW

File: devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/directive-explorer.component.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ import {
   ElementPropertyResolver,
   FlatNode as PropertyFlatNode,
 } from './property-resolver/element-property-resolver';
+import {PropertyTabComponent} from './property-tab/property-tab.component';
+import {SplitAreaDirective} from '../../vendor/angular-split/lib/component/splitArea.directive';
+import {SplitComponent as SplitComponent_1} from '../../vendor/angular-split/lib/component/split.component';
```
Comment (AleksanderBodurri) [inline devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/directive-explorer.component.ts:47]: Do we need to alias here? I notice this done in a few other places as well

File: devtools/projects/ng-devtools/src/lib/devtools-tabs/router-tree/router-tree.component.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import {Events, MessageBus, Route} from 'protocol';
   selector: 'ng-router-tree',
   templateUrl: './router-tree.component.html',
   styleUrls: ['./router-tree.component.scss'],
+  standalone: true,
```
Comment (AleksanderBodurri) [inline devtools/projects/ng-devtools/src/lib/devtools-tabs/router-tree/router-tree.component.ts:17]: Tough to say for sure since we don't use this component currently but I noticed the module version of this imported MatDialogModule and MatSelectModule. Do we need those here (The standalone equivalents)?

File: devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/property-tab/property-view/property-editor.component.ts [hunk 3]
```diff
@@ -17,6 +17,7 @@ import {
   Output,
 } from '@angular/core';
 import {ContainerType} from 'protocol';
+import {FormsModule} from '../../../../../../../../../packages/forms/src/form_providers';
```
Comment (AleksanderBodurri) [inline devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/property-tab/property-view/property-editor.component.ts:20]: I noticed we do these long relative imports in a few places in this PR. Are we not able to import these with `@angular/<package-name>` ?

File: devtools/src/app/devtools-app/devtools-app.module.ts [hunk 4]
```diff
@@ -14,7 +14,6 @@ import {DevToolsModule as NgDevToolsModule} from 'ng-devtools';
 import {DevToolsComponent} from './devtools-app.component';
```
Comment (AleksanderBodurri) [inline devtools/src/app/devtools-app/devtools-app.module.ts:14]: Can we remove the `import {DevToolsModule as NgDevToolsModule} from 'ng-devtools';` above this

File: devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/property-tab/property-view/property-editor.component.ts (hunk 3 above)
Comment (JeanMeche) [reply devtools/projects/ng-devtools/src/lib/devtools-tabs/directive-explorer/property-tab/property-view/property-editor.component.ts:20]: Good catch, those were generated by the migration schematics !

File: devtools/projects/ng-devtools/src/lib/devtools-tabs/router-tree/router-tree.component.ts (hunk 2 above)
Comment (JeanMeche) [reply devtools/projects/ng-devtools/src/lib/devtools-tabs/router-tree/router-tree.component.ts:17]: The component template doesn't use any component, so it seem we're good !

File: packages/core/src/di/r3_injector.ts [hunk 5]
```diff
@@ -482,7 +482,9 @@ function injectableDefOrInjectorDefFactory(token: ProviderToken<any>): FactoryFn
   }
 
   // There was no way to resolve a factory for this token.
-  throw new RuntimeError(RuntimeErrorCode.INVALID_INJECTION_TOKEN, ngDevMode && 'unreachable');
+  throw new RuntimeError(
```
Comment (AleksanderBodurri) [inline packages/core/src/di/r3_injector.ts:485]: This seems not necessarily related to DevTools. Lets put this change in another PR

**Review by AleksanderBodurri — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
> Migrated with the schematics and cleanup by hand

@JeanMeche any feedback / possible improvements to the migration schematic so we can minimize the "by hand" operations?

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 93845373a3ad7a5b67b5e6762fbe29f7b136c140.
